presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java

```diff
@@ -659,24 +659,24 @@ public void finishCreateTable(ConnectorSession session, ConnectorOutputTableHand
         return missingFileNames;
     }
 
-    private void createEmptyFile(HiveWritableTableHandle tableHandle, Path targetPath, Table table, Optional<Partition> partition, List<String> fileNames)
+    private void createEmptyFile(Path path, Table table, Optional<Partition> partition, List<String> fileNames)
     {
-        JobConf conf = new JobConf(hdfsEnvironment.getConfiguration(targetPath));
+        JobConf conf = new JobConf(hdfsEnvironment.getConfiguration(path));
         boolean compress = HiveConf.getBoolVar(conf, COMPRESSRESULT);
 
         Properties schema;
         String outputFormat;
         if (partition.isPresent()) {
             schema = getHiveSchema(partition.get(), table);
-            outputFormat = tableHandle.getPartitionStorageFormat().getOutputFormat();
+            outputFormat = partition.get().getStorage().getStorageFormat().getOutputFormat();
```

Comment:
Is this a bug in the existing code or just a simplification?  Either way, I'd rewrite the commit message to make it clear what is being changed.